config/install/akamai.settings.yml

```diff
@@ -0,0 +1,12 @@
+akamai_disabled: 1
+akamai_restapi_endpoint: https://api.ccu.akamai.com/ccu/v2/queues/default
+akamai_basepath: null
+akamai_timeout: 5
+akamai_username: null
+akamai_password: null
+akamai_domain:
+  - staging
```

Comment:
I don't think "Action" or "Domain" is a sequence, I think its a string selection of one of these. We can set a sane default upon install at your discretion